source/daplink/interface/swd_host.c

```diff
@@ -714,6 +733,48 @@ uint8_t swd_flash_syscall_exec(const program_syscall_t *sysCallParam, uint32_t e
     return 1;
 }
 
+#ifdef TARGET_MCU_PSOC6
+uint8_t swd_flash_verify_syscall_exec(const program_syscall_t *sysCallParam, uint32_t entry, uint32_t arg1, uint32_t arg2, uint32_t arg3, uint32_t arg4)
```

Comment:
What makes this function specific to PSoC6? Can it be made generic?

Reply:
I propose a slightly different solution: an additional parameter for the swd_flash_syscall_exec function to handle error level or first address of the comparison error returned by FlashAlgo.

Reply:
code has been redesigned to be generic.